pennylane_qiskit/converter.py

```diff
@@ -175,7 +175,7 @@ def _function(params: dict = None, wires: list = None):
                 # variable from the var_ref_map dictionary.
 
                 parameters = []
-                for p in op[0].params:
+                for p in op[0].data:
```

Comment:
Will have to stay as is, since here `op[0]` is a Qiskit gate :)
```suggestion
                for p in op[0].params:
```